classes/genre.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './music_album'
+
+class Genre
+  attr_reader :id
+  attr_accessor :name, :items
+
+  def initialize(name, id: Random.rand(1..1000))
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre << self
```

Comment:
According to the project requirements:

- [ ] Implement methods:

- add_item method in the Genre class
- should take an instance of the Item class as an input
- should add the input item to the collection of items
- should add self as a property of the item object (by using the correct setter from the item object) 
- I think it would be:
```suggestion
  def add_item(item)
    unless item.genre == self
      @items << item
      item.genre = self
    end
```

Reply:
🤔